cli/commands/project_create.go

```diff
@@ -0,0 +1,71 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/phase2/rig/cli/util"
+	"github.com/urfave/cli"
+)
+
+type ProjectCreate struct {
+	BaseCommand
+}
+
+func (cmd *ProjectCreate) Commands() []cli.Command {
+	create := cli.Command{
+		Name:        "create",
+		Aliases:     []string{},
+		Usage:       "Run a yeoman generator in a container to create project configuration files",
```

Comment:
Can we change this to something like "Run a code generator to create new project scaffolding."

There's code as well as configuration in generator output, and I'm not sure we need to specify yeoman/container in the short description.